src/main/webapp/resources/js/pages/projects/share/TargetMetadataRestriction.jsx

```diff
@@ -17,10 +18,15 @@ import { updateMetadataRestriction } from "./shareSlice";
  *  project, if it is not in the target project it get the current restriction.
  * @param {array} restrictions - list of available restrictions
  * @param {function} onChange - change handler
+ * @param {boolean} newProject - if a new project is being created or not
  * @returns {JSX.Element}
  * @constructor
  */
-export function TargetMetadataRestriction({ field = {}, restrictions = [] }) {
+export function TargetMetadataRestriction({
```

Comment:
Since you are repurposing this component for another page, maybe we should move it into the components directory.